### DISCUSSION TITLE
Sending syslog from pfsense without agent

### DISCUSSION THREAD

**Discussion by anitatata** [Security-Onion-Solutions/securityonion#12245] [2.4] [answered]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

cloud

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

20

### Storage for /

100

### Storage for /nsm

500

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello, I just installed security onion version 2.4.30.
Everything working fine, I already added some elastic agent (in linux based VM). All log is send and we can see it in security onion.
But there is one problem with pfsense, so I just follow this step
https://docs.securityonion.net/en/2.4/pfsense.html#
I try with the simple parser, but the log from pfsense is not coming to security onion.

I try to check the syslog from security onion, but there is no log from the pfsense
<img width="960" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/dbfef23b-1e12-4e14-92e9-045892f2b470">

And also I try to check syslog in elastic, still no log
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/246cfe18-03b8-43a4-b44f-7e907bde7b15)

Please kindly help, if there is miss configuration.
Thanks before

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
Did you allow the IP in the Firewall?

**Reply by anitatata** [reply]
Yes, I already allow the ip in the Firewall

**Comment by TotieBash** [comment, marked as answer]
#12055 check the fix.   I use the Elastic integrations https://docs.securityonion.net/en/2.4/pfsense.html#elastic-integration-for-pfsense

Run the following commands below from the SO Standalone.  Send us the screenshots results.
ss -tulpn | grep 9001
iptables --list -n | grep 9001

**Reply by anitatata** [reply]
Thanks for the answer.

I try to follow the instruction
- add integration
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/998922df-6f94-491a-a3cb-6e62688a319a)


![image](https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/c9275f69-5e38-4bef-982d-e1df399114ba)


But I did not understand, this statement 
<img width="602" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/1934fdc0-afcf-4691-bee4-09788870ccfe">
Is it means like this?
<img width="825" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/f86f6b22-e568-482d-a6be-632cdba42c6d">


For the kibana dashboard, I don't see log from pfsense integration
<img width="960" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/e1ddc3f4-bd6b-484a-9f0d-4fdd240dc1cc">


Please kindly help.
Thanks before

**Reply by TotieBash** [reply]
We're close...  The Elastic Integration looks good...  

So it is the firewall that is the issue.  So since we are both using SO Standalone your firewall path should be:
### Firewall > role > standalone > chain > INPUT > hostgroups > customhostgroup0 > portgroups
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/86748904/805ae511-5da1-43fe-bca2-9be69e5cbd14)

It should look like this
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/86748904/c89189e5-3aae-4718-84c0-0e6115bbb22c)

click "SYNCHRONIZE GRID" to force salt apply the changes.  This might take 2-3minutes to run in the background

then issue "iptables --list -n | grep 9001" and "iptables --list -n | more" it should look like below.  Mine is using port 9002
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/86748904/4a264165-f4fc-4d90-a650-c15804b48e79)

**Reply by anitatata** [reply]
Thanksss, finally my pfsense can connect and send the log.
In my pfsense, I change the log message format
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/a0e3d914-9f68-4740-9a9f-10b0e547dfc1)

<img width="960" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/13854947/fe2a5f14-d762-4047-992c-8b0d861fa498">

Thank you so much

**Reply by anitatata** [reply]
Anyway, If I have a switch device, what integrations can I use?

**Reply by TotieBash** [reply]
What kind of switch? if it is Cisco IOS then its in the latest 2.4.40 patch..
